ingest-app/resources/granule_bulk_update_schema.json

```diff
@@ -6,18 +6,29 @@
         "UPDATE_FIELD"
       ]
     },
+    "UpdateFieldEnum": {
+      "type": "string",
+      "enum": [
+        "OPeNDAPLink",
+        "S3Link"
+      ]
+    },
+    "UpdateTupleType": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "type": "string"
+      },
+      "examples": [
+        ["SL:AB_5DSno.008:30500511", "s3://aws.example.com"]
```

Comment:
The examples should be consistent among the fields. We shouldn't use `OPeNDAPLink` in Operation, but an S3 link in the url value.